docs/customresourcestate-metrics.md

```diff
@@ -168,8 +168,8 @@ spec:
 Produces the following metrics:
 
 ```prometheus
-kube_crd_active_count{group="myteam.io", kind="Foo", version="v1", active="1",custom_metric="yes",foo="bar",name="foo",bar="baz",qux="quxx",type="type-a"} 1
-kube_crd_active_count{group="myteam.io", kind="Foo", version="v1", active="3",custom_metric="yes",foo="bar",name="foo",bar="baz",qux="quxx",type="type-b"} 3
+kube_crd_active_count{group="myteam.io", kind="Foo", version="v1", active="1",custom_metric="yes",foo="bar",name="foo",bar="baz",qux="quxx",type="type-a"} 2
```

Comment:
While at it, can you also fix the generated metric name here? The example above says `ready_count` and not `active_count`.